pg-ragtime/src/pg/ragtime.clj

```diff
@@ -0,0 +1,29 @@
+(ns pg.ragtime
+  (:require
+   [ragtime.core :as ragtime]
+   [ragtime.strategy]
+   [ragtime.protocols]
+   [pg.core :as pg])
+  (:import org.pg.Connection))
+
+(defn ensure-migrations-table!
+  [conn]
+  (pg/query conn "
+create table if not exists migrations(
+  id text primary key,
+  timestamp timestamp default current_timestamp
+)"))
```

Comment:
I'm uncertain about the name of the migrations table.

In my own code, I wanted the name `migrations`. I think a better default here is `ragtime_migrations`. A bit more specific.

It could also make sense that the user has to choose the name of the migrations table. In that case, perhaps the table name could be provided as config to `pg.core/connect`.